source/codegen/templates/service.cpp.mako

```diff
@@ -110,6 +125,9 @@ namespace ${config["namespace_component"]}_grpc {
     response_type = service_helpers.get_response_type(method_name)
     is_async_streaming = common_helpers.has_async_streaming_response(function_data)
 %>\
+% if function_name in streaming_functions:
```

Comment:
This really should be an `elif function_name in streaming_functions_to_generate` before the last `else` which calls `define_simple_method_body` and then change your helper from `define_streaming_api` to `define_streaming_api_body`. Most of your implementation in `define_streaming_api` after you remove `struct` and moniker methods from there is same as non-async function definition in `else` block starting at line 122/140 below.

Reply:
Updated.